tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1874,6 +1874,13 @@ def rgb_to_grayscale(images, name=None):
 
   Returns:
     The converted grayscale image(s).
+    
+  Usage Example:
+   ```python
+   import tensorflow as tf
+   original = tf.random.normal(shape=(256, 256, 3))
+   converted = tf.image.rgb_to_grayscale(original)
```

Comment:
Can you choose a sufficiently small image with nice floating point values (a 1x1 image would work here, for example) so we can make this a doctest?

Reply:
Ok, shall I replace line 1881 with `original = tf.random.normal(shape=(1, 1, 3))` instead? Default output value should already be tf.dtypes.float32, would that need to change as well? Thanks for the feedback.